src/main/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbByTable.java

```diff
@@ -77,8 +77,6 @@ private MtAmazonDynamoDbByTable(MtAmazonDynamoDbContextProvider mtContext, Amazo
     }
 
     /**
-     * Determines if the table for the given name is a multitenant table associated with this instance.
```

Comment:
any reason to delete this doc (or the yet-to-be-fleshed-out docs below)?

Reply:
this doc is [almost] literally identical to the snippet after @return, useless IMO. 

Also, the javadocs `TODO: write Javadoc.` bothers/annoys me almost physically, maybe that's my OCD.

Reply:
pulled the whole snippet out, already javadoc-ed in super class